### ISSUE TITLE
Improve description for Milestone items

### ISSUE THREAD

**Issue by leemun1** [microsoft/vscode#121610]
Hello, I am new to this repo and am looking for ways I could contribute.

I am a little confused about the categorization of the issues however.
I read from the "How to Contribute" page that issues labeled as `Good first issue` or `Helped wanted` are usually good ones for fresh contributors to work on.

However, some of these issues are also assigned to specific Milestones, like below:

![image](https://user-images.githubusercontent.com/17537040/115167655-61b13f80-a086-11eb-994d-1599d275f9ee.png)

When I go to the Milestones page to read more about each Milestone, the existing descriptions for some gives the impression that certain Milestones are reserved for community contribution whereas some ones aren't.

For example, see below:

![image](https://user-images.githubusercontent.com/17537040/115167482-c4560b80-a085-11eb-800b-fbcb69f61858.png)

After reading the descriptions for the `Backlog` and `Backlog Candidates` Milestones, my understanding is that `Backlog Candidates` are expecting community contribution whereas `Backlog` isn't. Is my understanding correct?

Basically, I think an outline of which issue category or which Milestone are available for community contributors to jump on, and which ones aren't would be super helpful. 

I apologize if this is already documented somewhere, or if I am totally misunderstanding something.
Any pointers or clarification would be greatly appreciated!

**Comment by alexr00** [comment]
Added https://github.com/microsoft/vscode/wiki/How-to-Contribute/_compare/ea9addfde7543612d1d2bf06e829a02b950b08c3

Basically any issue with `good-first-issue` or `help-wanted` is an issue we're seeking community contribution from. The Milestones are mostly for our own bookkeeping.
